src/phoenix/datasets/evaluators/code_evaluators.py

```diff
@@ -75,18 +56,14 @@ def evaluate(self, exp_run: ExperimentRun, example: Example) -> EvaluationResult
         )
 
 
-class ContainsAllKeywords:
-    annotator_kind = AnnotatorKind.CODE.value
-
+class ContainsAllKeywords(Evaluator):
     def __init__(self, keywords: List[str], name: Optional[str] = None) -> None:
         self.keywords = keywords
-        self.name = name or f"ContainsAll({keywords})"
+        self._name = name or f"ContainsAll({keywords})"
 
-    def evaluate(self, exp_run: ExperimentRun, example: Example) -> EvaluationResult:
-        assert exp_run.output is not None
-        result = _unwrap_json(exp_run.output.result)
-        assert isinstance(result, str), "Experiment run output must be a string"
-        not_found = [keyword for keyword in self.keywords if keyword not in result]
+    def evaluate(self, *, output: Optional[TaskOutput] = None, **_: Any) -> EvaluationResult:
+        assert isinstance(output, str), "Experiment run output must be a string"
+        not_found = [keyword for keyword in self.keywords if keyword not in output]
```

Comment:
same as the above. I get the pitfall of it not being a string but what if I just want to see if a JSON payload contains keywords?